paper.tex

```diff
@@ -80,17 +81,19 @@ \subsection{Cycle}
 This analysis is focused only on the second strata of EG29, represented by a
 single PWR reactor using MOX fuel. After irradiation the uranium and the
 plutonium are separated and reprocessed to fuel the next batch of MOX fuel for
-the PWR. The separation efficiency is $0.988$. The separated U/Pu (``J1''
-stream) is then separated into two sub-stream, one used for new PWR deployment
-(``J1\_prime\_storage'' on Fig. \ref{fig:flow}) and the other one for the
-production the new batch of MOX fuel (``J1\_second\_storage''). The respective
-fraction of those sub-stream are $0.105$ and $0.895$.
+the PWR. As no loss are associated to the fabrication process, the separation
+efficiency is set at $0.988$, which corresponding to the $1\%$ loss in
+separation and the $0.2\%$ during fabrication as specified in \cite{FCDP}).
+About $10\%$ of this U/Pu stream (``J1`` stream) is arbitrary set aside in order
+to support an hypothetical new PWR deployment (``J1\_prime\_storage), the
+remaining $90\%$ is used for the production the new batch of MOX fuel
+(``J1\_second\_storage'').
```

Comment:
Maybe you can add a sentence like, "Analyzing the precise value of these quantities is not the focus of this work, and many possible values could be used to demonstrate the outcomes shown here."